actors/runtime/src/actor_error.rs

```diff
@@ -1,6 +1,44 @@
 use fvm_shared::error::ExitCode;
 use thiserror::Error;
 
+mod abort {
+    use crate::ActorError;
+    use fvm_shared::error::ExitCode;
+
+    #[derive(thiserror::Error, Debug)]
+    #[error("abort error")]
+    pub struct Abort {
+        /// This ensures that this error can not be crated outside.
+        _private: (),
+    }
```

Comment:
We can just make it impossible to construct:

```suggestion
    pub enum Abort {}
```

Reply:
nice